Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -638,6 +643,62 @@ int CLuaElementDefs::OOP_GetElementVelocity ( lua_State* luaVM )
     return 1;
 }
 
+int CLuaElementDefs::GetElementTurnVelocity ( lua_State* luaVM )
+{
+    // Verify the argument
+    CClientEntity* pEntity = NULL;
+    CScriptArgReader argStream ( luaVM );
+    argStream.ReadUserData ( pEntity );
+
+    if ( !argStream.HasErrors () )
+    {
+        // Grab the element, verify it
+        if ( pEntity )
```

Comment:
"verifications" are done by `CScriptArgReader` now, so you don't need to manually do LogBadPointer.